modules/SofaHaptics/SofaHaptics_test/scenes/ToolvsFloorCollision_test.scn

```diff
@@ -1,13 +1,8 @@
 <?xml version="1.0" ?>
 <Node name="root" dt="0.05" showBoundingTree="0" gravity="0 -1 0">
-    <RequiredPlugin name="SofaOpenglVisual"/>
+    <RequiredPlugin name="Sofa.Component" />
```

Comment:
I see you lazy boy :eyes:

Reply:
🫢 
it is a test so I dont really bother